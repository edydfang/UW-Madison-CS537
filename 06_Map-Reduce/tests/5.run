./wordcount tests/9.txt 1 1 1 > tests-out/5.mid; python3 wordcount_parser.py --input-file tests-out/5.mid