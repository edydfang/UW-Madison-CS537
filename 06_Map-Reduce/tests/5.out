Venus 1
abuse 1
adventure 1
ancestor 1
angel 1
arm 1
beg 1
bolt 1
branch 1
breakin 1
cable 1
cap 1
cellar 1
choke 1
colleague 1
conductor 1
core 1
create 1
dangerous 1
deficiency 1
despise 1
ego 1
establish 1
evolution 1
experience 1
experiment 1
explosion 1
fat 1
fight 1
final 1
frog 1
fruit 1
gas 1
gaspedal 1
global 1
guerrilla 1
gun 1
halt 1
healthy 1
hit 1
home 1
hypnothize 1
influence 1
information 1
instruction 1
jelly 1
jewel 1
jurisdiction 1
kid 1
knot 1
lane 1
launch 1
lost 1
minister 1
nap 1
national 1
noble 1
paint 1
parachute 1
paragraph 1
penalty 1
performer 1
philosophy 1
pick 1
pile 1
pioneer 1
pipe 1
pneumonia 1
porter 1
positive 1
program 1
pursuit 1
quit 1
rage 1
refrigerator 1
refuse 1
resort 1
rider 1
sanctuary 1
satellite 1
sensitive 1
simplicity 1
slot 1
snack 1
soil 1
stadium 1
summit 1
thesis 1
tiger 1
tin 1
tread 1
trip 1
twist 1
unlike 1
valid 1
van 1
variant 1
volume 1
weight 1
worm 1
